src/index.ts

```diff
@@ -400,22 +595,30 @@ export default class OneSignal {
         RNOneSignal.setInAppMessageLifecycleHandler();
     }
 
-    // Pass a String key and any value and creates a trigger map to pass to addTriggers()
-    static addTrigger(key, value) {
+    /**
+     * Add an In-App Message Trigger.
+     * @param  {string} key
+     * @param  {string} value
+     * @returns void
+     */
+    static addTrigger(key: string, value: string): void {
         if (!isObjectNonNull(RNOneSignal)) return;
 
         if (!key || !value) {
             console.error("OneSignal: addTrigger: must include a key and a value");
         }
 
-        let trigger = {};
+        let trigger: { [key: string]: string } = {};
         trigger[key] = value;
         RNOneSignal.addTriggers(trigger);
     }
 
-
-    // Expected format is Map<String, Object>, make sure all values are Objects and keys are Strings
-    static addTriggers(triggers) {
+    /**
+     * Adds Multiple In-App Message Triggers.
+     * @param  {object} triggers
+     * @returns void
+     */
+    static addTriggers(triggers: object): void {
```

Comment:
type definition can be made more specific (see comment above re: `addTrigger`)

Reply:
Done.. type is `{ [key: string]: string | number | boolean }`